TUTORIAL.md

```diff
@@ -0,0 +1,100 @@
+# Getting started
+Before you start, make sure you understand the concept or database migration and rollback.
+
+Create new Go project and configure a database for it. Make sure that your database is supported [here](README.md#databases)
```

Comment:
Migrate can be used to manage migrations with non-Go projects via the CLI, so creating a Go project isn't necessary.

Reply:
Fair enough. Is it worth adding it to FAQ?

Reply:
My guess is that migrate is primarily used with Go projects today, but that doesn't have to be the case. No one has asked the question yet, but we could add it to the FAQ 😄 

e.g. 
**Can I use `migrate` with a non-Go project?**
Yes, you can use the `migrate` CLI in a non-Go project, but there are probably other libraries/frameworks available that offer better test and deploy integrations in that language/framework.

Reply:
Let's assume I asked it three times recently :D but honestly, I think it will not be harmful to add it there ;)